metricbeat/module/kubernetes/apiserver/_meta/docs.asciidoc

```diff
@@ -1 +1,32 @@
-This is the `apiserver` metricset of the Kubernetes module.
+This is the `apiserver` metricset of the Kubernetes module, in charge of retrieving metrics from the Kubernetes API (available at `/metrics`).
+
+This metricset needs access to the `apiserver` component of Kubernetes, accessible typically by any POD via the `kubernetes.default` service or via environment variables (`KUBERNETES_SERVICE_HOST` and `KUBERNETES_SERVICE_PORT`).
+
+When the API uses https, the pod will need to authenticate using its default token and trust the server using the appropiate CA file.
+
+Configuration example using https and token based authentication:
+
+```yaml
+    - module: kubernetes
```

Comment:
Can you we fix the indentation here? It should be one shift left.

Reply:
pushed and solved

Reply:
ops... I forgot the docs update.... wait for it :)